proposals/0429-partial-consumption.md

```diff
@@ -4,7 +4,7 @@
 * Authors: [Michael Gottesman](https://github.com/gottesmm), [Nate Chandler](https://github.com/nate-chandler)
 * Review Manager: [Xiaodi Wu](https://github.com/xwu)
 * Implementation: On `main` gated behind `-enable-experimental-feature MoveOnlyPartialConsumption`
```

Comment:
Should this line be deleted or replaced with a link to https://github.com/swiftlang/swift/pull/73159 ?

Reply:
Ah, yes, we can just remove the implementation field